### ISSUE TITLE
ApplicationId is required on Typescript types while it should not

### ISSUE THREAD

**Issue by soullivaneuh** [Vonage/vonage-node-sdk#277]
According to the documentation (and the example from your dashboard):

> (optional) The Nexmo Application ID to be used when creating JWTs.

But the option is declared as required here:

https://github.com/Nexmo/nexmo-node/blob/58935f0baf6c76f61f82717da883bd62853543d1/typings/index.d.ts#L6

AFAIK, it should be `applicationId?: string;`.

**Comment by AlexLakatos** [comment]
Yes, you're right. Thanks for submitting the issue.

Fixed in #280 and will be released in v2.5.2 next week.

**Comment by soullivaneuh** [comment]
Thanks for the reactivity, I appreciate it! :+1:

**Comment by soullivaneuh** [comment]
For other people willing into the issue, simply add en empty string at this option as a workaround:

```typescript
const nexmo = new Nexmo({
  // @see https://github.com/Nexmo/nexmo-node/issues/277
  applicationId: '',
  apiKey: process.env.NEXMO_API_KEY || 'none',
  apiSecret: process.env.NEXMO_API_SECRET || 'none',
});
```

**Comment by AlexLakatos** [comment]
This has been fixed and released in nexmo@2.5.2
